dub.json

```diff
@@ -4,5 +4,9 @@
     "homepage": "https://github.com/DlangGraphicsWG/",
     "license": "BSL-1.0",  
     "sourcePaths": [ "src" ],
-    "importPaths": [ "src" ]
+    "importPaths": [ "src" ],
+    "targetPath" : "build",
+    "targetType": "executable",
+    "dflags-ldc" : ["-finstrument-functions"]
```

Comment:
Is this needed?

Reply:
It is not necessary. I added it so I can profile the code using ldc compiler. I guess it could be useful to all of us while developing everything, but I can remove it you want...

Reply:
Please do and then I'll +1.